src/gpu/multiexp/ec.cl

```diff
@@ -41,65 +44,70 @@ POINT_projective POINT_double(POINT_projective inp) {
 
 // http://www.hyperelliptic.org/EFD/g1p/auto-shortw-jacobian-0.html#addition-madd-2007-bl
 POINT_projective POINT_add_mixed(POINT_projective a, POINT_affine b) {
-  if(b.inf) return a;
+  if(b.inf) {
+      return a;
+  }
 
-  if(FIELD_eq(a.z, FIELD_ZERO)) {
+  const FIELD tmp_Zero = FIELD_ZERO;
+  if(FIELD_eq(a.z, tmp_Zero)) {
+    FIELD tmp = FIELD_ONE;
     a.x = b.x;
     a.y = b.y;
-    a.z = FIELD_ONE;
+    a.z = tmp;
```

Comment:
same here, let's pick a better name for all these `tmp` variables.

Reply:
done